registry/registry.go

```diff
@@ -99,7 +101,12 @@ func doRequest(req *http.Request, jar http.CookieJar, timeout TimeoutType, secur
 			return false
 		}
 
-		hostDir := path.Join("/etc/docker/certs.d", req.URL.Host)
+		if runtime.GOOS == "windows" {
+			hostDir = filepath.Join(os.TempDir(), "/docker/certs.d")
```

Comment:
why do we store certs at a temp dir? `/etc/docker/certs.d` is persistent at linux.